crates/sidevm/env/src/args_stack.rs

```diff
@@ -171,7 +171,9 @@ impl<'a, A> ArgDecode<'a, A> for &'a [u8] {
         Self: Sized,
     {
         let ((len, ptr), stack) = stack.pop();
-        Ok((vm.slice_from_vm(ptr, len)?, stack))
+        let bytes = vm.slice_from_vm(ptr, len)?;
+        let bytes = unsafe { core::mem::transmute(bytes) };
```

Comment:
Why do we use a lot of transmute here? Let's add some comment?

Reply:
I finally managed to avoid those transmutes because the `wasmer::MemoryView<'a>` doesn't bind its lifetime to the `Store` but to its `Handle`(This might be a mistake in wasmer, it should bind to the Store).
Although the transmute was removed, the memory safety guarantee doesn't change. We still need to ensure the `MemoryView` doesn't outlive the Store by hand.